spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/SpatialBatchNormalization.scala

```diff
@@ -94,6 +94,8 @@ class SpatialBatchNormalization(
 
     runningMean.zero()
     runningVariance.zero()
+
+    gradWeightAndBias.zero()
```

Comment:
do not check affine?

Reply:
This BatchNorm only support the true of affine.